addons/essentials/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import { logger } from '@storybook/node-logger';
+import dedent from 'ts-dedent';
+
+type PresetOptions = {
+  actions?: any;
+  backgrounds?: any;
+  docs?: any;
+  knobs?: any;
+  links?: any;
+  viewport?: any;
+};
+
+let packageJson: any = {};
+if (fs.existsSync('./package.json')) {
+  try {
+    packageJson = JSON.parse(fs.readFileSync('./package.json').toString());
+  } catch (err) {
+    logger.error(`Error reading package.json: ${err.message}`);
+  }
+}
+
+const warnIfInstalled = (addon: string, dependencies?: Record<string, string>) => {
+  if (dependencies) {
+    const version = dependencies[addon];
+    if (version) {
+      logger.warn(dedent`
```

Comment:
We should log a recommendation in the warning, e.g "we recommend you remove it as a dependency of your app and allow addon-essentials to include the right version instead".